src/libraries/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/ActivityContext.cs

```diff
@@ -14,16 +14,18 @@ namespace System.Diagnostics
         /// <summary>
         /// Construct a new object of ActivityContext.
         /// </summary>
-        /// <param name="traceId"> A trace identifier.</param>
-        /// <param name="spanId"> A span identifier </param>
+        /// <param name="traceId">A trace identifier.</param>
+        /// <param name="spanId">A span identifier.</param>
         /// <param name="traceFlags">Contain details about the trace.</param>
-        /// <param name="traceState"> Carries system-specific configuration data.</param>
-        public ActivityContext(ActivityTraceId traceId, ActivitySpanId spanId, ActivityTraceFlags traceFlags, string? traceState = null)
+        /// <param name="traceState">Carries system-specific configuration data.</param>
+        /// <param name="isRemote">Indicate the context is propagated from remote parent.</param>
```

Comment:
should we leave a comment in the ActivityContext summary that this is inspired from OT? W3C spec has nothing like `IsRemote`, so it may hint users where/what is the motivation for having IsRemote

Reply:
sure. I'll add the comment